### ISSUE TITLE
Is the referrer_id in Woo supposed to point towards the MRN or the System ID in Hexly?

### ISSUE THREAD

**Issue by jonsheppard** [hexly/backoffice#60]
**TL;DR Orders seem to be getting assigned to the wrong person after the MRN/System ID changes we made last week.**

Team member Rani Bivens (MRN: 7345, Sys ID: 49584) has 11 February orders in Hexly but only 2 in WooCommerce. The orders showing in Hexly are: 53280, 53409, 53512, 53548, 53563, 53783, 53810, 54291, 54565, 54567, 54853. WooCommerce only shows orders 53512, 54291

Now I see an issue where the orders are assigning the System ID instead of the MRN and people are saying that the orders have disappeared from their backoffice now. (see order 54896 in Woo, referrer_id 49520)

The original support request from Rani:

"I had two people order under me the end of this week, I saw both orders under sales Friday. Both ordered the 1000 mg oils. Neither are showing up now. Can you please check on this. Their names are Jamie Thomas and Kendra Garrett. Thank you!"
